udemy-reviews-producer/src/main/resources/avro/udemy-review.avsc

```diff
@@ -3,7 +3,6 @@
   "type": "record",
   "name": "User",
   "fields": [
-    {"name": "id", "type": "long", "doc": "User ID in Udemy's DB"},
```

Comment:
user node no longer reveals their ID. Perhaps it was a security issue, know knows.